.npmrc

```diff
@@ -0,0 +1 @@
+lockfile-version=2
```

Comment:
This needs to be added to the repo manually since grunt-contrib-internal doesn't add it